packages/@vue/cli-ui/src/components/app/ProjectQuickDropdown.vue

```diff
@@ -1,84 +1,98 @@
 <template>
-  <VueDropdown
-    v-if="$responsive.wide"
-    :label="projectCurrent ? projectCurrent.name : $t('org.vue.components.status-bar.project.empty')"
-    class="current-project project-quick-dropdown"
-    icon-right="arrow_drop_down"
-    button-class="round"
-  >
-    <!-- Current project options -->
-
-    <template v-if="projectCurrent">
-      <VueSwitch
-        :value="projectCurrent.favorite"
-        :icon="projectCurrent.favorite ? 'star' : 'star_border'"
-        class="extend-left"
-        @update="toggleCurrentFavorite()"
-      >
-        {{ $t('org.vue.components.project-select-list-item.tooltips.favorite') }}
-      </VueSwitch>
+  <div class="project-quick-dropdown">
+    <VueDropdown
+      v-if="$responsive.wide"
+      :label="projectCurrent ? projectCurrent.name : $t('org.vue.components.status-bar.project.empty')"
+      class="current-project"
+      icon-right="arrow_drop_down"
+      button-class="round"
+    >
+      <!-- Current project options -->
+
+      <template v-if="projectCurrent">
+        <VueSwitch
+          :value="projectCurrent.favorite"
+          :icon="projectCurrent.favorite ? 'star' : 'star_border'"
+          class="extend-left"
+          @update="toggleCurrentFavorite()"
+        >
+          {{ $t('org.vue.components.project-select-list-item.tooltips.favorite') }}
+        </VueSwitch>
+
+        <VueDropdownButton
+          :label="$t('org.vue.components.project-select-list-item.tooltips.open-in-editor')"
+          icon-left="open_in_browser"
+          @click="openInEditor(projectCurrent)"
+        />
+
+        <VueDropdownButton
+          :label="$t('org.vue.components.project-rename.title')"
+          icon-left="local_offer"
```

Comment:
This icon typically indicates "label" rather than "rename" IMO.
Actually I failed to recognize it in the first glance…

Reply:
Maybe a pen then? https://material.io/tools/icons/?icon=create&style=baseline

Reply:
Looks good

Reply:
Could you replace the two `local_offer` with `edit` then? Thanks!